pkg/bootstrap/docker/up.go

```diff
@@ -755,7 +755,7 @@ func (c *ClientStartConfig) Factory() (*clientcmd.Factory, error) {
 }
 
 // Clients returns clients for OpenShift and Kube
-func (c *ClientStartConfig) Clients() (*client.Client, *kclient.Client, error) {
+func (c *ClientStartConfig) Clients() (client.Interface, *kclient.Client, error) {
```

Comment:
@liggitt Done. But there's some kubernetes methods that use Client instead of Interface so need type assertion... like [here](https://github.com/guilhermebr/origin/blob/bb41e0ae73a00552691ef60fe39482c06fefa400/pkg/cmd/cli/cmd/idle.go#L536):